src/plugins/intel_npu/src/plugin/npuw/lazy_tensor.cpp

```diff
@@ -19,15 +21,28 @@ namespace npuw {
 namespace weights {
 namespace op {
 struct Const {
-    std::shared_ptr<ov::op::v0::Constant> m_node;
+    std::shared_ptr<ov::op::v0::Constant> m_node = nullptr;
     ov::element::Type m_cached_type;
     ov::Shape m_cached_shape;
     const void* m_cached_ptr = nullptr;
+    std::size_t m_offset = 0;
+    std::size_t m_byte_size = 0;
+    ov::Tensor m_read_from_weights;
```

Comment:
probably `m_read_from_bin` is a better name, assuming we only read weightless weights from `.bin` (and do not support other cases)

Reply:
Done